### PR TITLE
Add the ./run command and tidy scripts

### PR DESCRIPTION
**Pull request by WillMoggridge**
Add the latest ./run command

# QA

Try the ./run commands locally from the readme

### CODE REVIEW

**Comment by Caleb-Ellis** [comment]
Build script seems to run three times: `npm-run-all build-css build-js`, `yarn install && yarn run build`, and `npm-run-all -p watch-css serve-storybook`. I think the only one we need is the last one, since it serves the Storybook app that we're developing on and also watches the CSS

**Comment by pmahnke** [comment]
- `./run clean` good
- `./run clear-cache` good
- `./run serve` took ages and perhaps finished (hard to know what to look for)?  also error on ctrl-c 

```
webpack built 62062d3e10c6d0b1299b in 38498ms
Storybook started on => http://localhost:8102/

webpack building...
webpack built 62062d3e10c6d0b1299b in 870ms

error Command failed with signal "SIGINT".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

- `./run build` good
- `./run test` good
- `./run stop` needs a command line arg

```
Stopping all running containers for canonical-webteam-vanilla-framework-f8df51b4
"docker kill" requires at least 1 argument.
See 'docker kill --help'.

Usage:  docker kill [OPTIONS] CONTAINER [CONTAINER...]
```

**Comment by WillMoggridge** [comment]
@pmahnke In the serve output, it start running with `Storybook started on => http://localhost:8102/`.
On my screen it is cyan colour but still not very obvious. However, I can't find a way to make that message more obvious. Webpack is watching for changes and will continue to build on every file change.

The error on ctrl+c is a slight annoyance but if I am right, it does not cause problems?
It looks like it can be fixed by loading sass files through webpack rather than using a separate job and having to run the two side by side. If it is OK, we could fix this after this PR.


I created an issue for the `.run stop` on the upstream repo: https://github.com/canonical-webteam/generator-canonical-webteam/issues/49

File: package.json [hunk 1]
```diff
@@ -4,28 +4,31 @@
...
+    "test": "node scripts/test.js --env=jsdom"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/vanilla-framework/vanilla-framework-react.git"
   },
+<<<<<<< HEAD
+=======
+  "author": "Canonical webteam <webteam@canonical.com>",
+>>>>>>> Update package.json author and URLs
```
Comment (Caleb-Ellis) [inline package.json]: @WillMoggridge I think maaaaaaaaybe this is why the CI is currently failing

Comment (WillMoggridge) [reply package.json]: Woah, what is with everyone putting me on blast?!

**Comment by WillMoggridge** [comment]
Tests are passing now

**Comment by Caleb-Ellis** [comment]
LGTM :+1:

**Pull request merged**